server/interfaces/interfaces.go

```diff
@@ -25,3 +27,11 @@ type Metrics interface {
 type StoreMetrics interface {
 	IncStoreOp(op string)
 }
+
+type Store interface {
+	Migrate(direction models.Direction, dryRun bool) error
+	Close() error
+	GetPost(postID string) (*model.Post, error)
+	UpdateFileInfoPostID(fileID, channelID, postID string) error
+	KVGet(pluginID, key string, fromWriter bool) ([]byte, error)
+}
```

Comment:
Hopefully we won't need this for much longer. It's currently used to allow mocking.